src/cdk/drag-drop/drag.ts

```diff
@@ -114,6 +120,12 @@ export class CdkDrag<T = any> implements OnDestroy {
    */
   private _moveEventSubscriptions = 0;
 
+  /** Keeps track of the direction in which the user is dragging along each axis. */
+  private _delta: {
```

Comment:
It's not immediately clear to me what `_delta.x` / `_delta.y` represents

Reply:
I'm not sure what else to call it. Something like `pointerDirection` isn't much better.

Reply:
Yeah, it's either 1 or -1, depending on whether the user is going up/down or left/right, similar to the `deltaX` and `deltaY` on the `WheelEvent`.

Reply:
How about splitting it up into two properties?
```ts
private _pointerDirectionDelta: {x: -1 | 1, y: -1 | 1};
private _pointerPositionAtLastDirectionChange: Point;
```

Reply:
Sounds good. Done.